### ISSUE TITLE
Validation of the simplest and minimal AAS is wrong

### ISSUE THREAD

**Issue by lenaxus** [admin-shell-io/aas-test-engines#6]
## Expected Behavior

I want to check my AASes to prove if they are Metamodel Version 3 confirm. I have choosen the variant to check json data directly. I had troubles when checking my AASes because it said all the time they are invalid. So I started with the simplest and minimal example. I created a new empty AAS in the AASX Package Explorer and exported the JSON of it. I expected that this AAS would be valid.

## Actual Behavior

Unfortunately the result of the checked AAS is that is invalid:

```
Check JSON
   Does not match all sub-schemas
      Reference #/$defs/Environment is invalid
         Property assetAdministrationShells is invalid
            item 0 is invalid
```

## Steps to Reproduce the Problem

  I used this script to check the AAS with the JSON directly in the python script:

```
from aas_test_engines import file

aas = {
    'assetAdministrationShells': [
      {
          "id": "https://example.com/ids/aas/0532_3101_1042_1688",
          "assetInformation": {
            "assetKind": "Instance"
          },
          "submodels": [],
          "modelType": "AssetAdministrationShell"
      }
    ]
}
result = file.check_json_data(aas)
result.dump()
```

**Comment by lenaxus** [comment]
**Update:**
I invested more time to find the problem. I checked [this minimal AAS](https://github.com/admin-shell-io/aas-specs/blob/master/schemas/json/examples/generated/AssetAdministrationShell/minimal.json) and it was valid. This has brought me further in the validation. 

The json of this minimal AAS is the following:
```
{
  "assetAdministrationShells": [
    {
      "assetInformation": {
        "assetKind": "NotApplicable",
        "globalAssetId": "something_eea66fa1"
      },
      "id": "something_142922d6",
      "modelType": "AssetAdministrationShell"
    }
  ]
}
```

I have two open questions about this validation:
1. If I delete the assetInformation.globalAssetId which is declared as optional in the v3 Metamodell specification the AAS gets invalid. Why is this the case when it is optional?
2. If I add to the upper AAS the following: `"submodels": []` it becomes invalid. Why is an empty array for submodels invalid?
